include/odp/api/spec/crypto_types.h

```diff
@@ -958,7 +959,8 @@ typedef struct odp_crypto_op_result {
  * Crypto packet API operation result
  */
 typedef struct odp_crypto_packet_result_t {
-	/** Request completed successfully */
+	/** Request completed successfully. True when odp_crypto_result()
+	 *  returned zero. This field will be deprecated in the future */
```

Comment:
Period (.) missing in the end.

Reply:
fixed as a side effect of squashing with the commit that deprecates the ok field